intl-config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  "dest": "public",
+  "src": "locales"
```

Comment:
This is the source of where we build the locales from, and where we put it.

Not sure why it needs a config for such a simple amount of info, but I guess some configs might be more complex.